drivers/bluetooth/hci/hci_silabs_efr32.c

```diff
@@ -21,16 +21,22 @@ struct hci_data {
 	bt_hci_recv_t recv;
 };
 
-#define SL_BT_CONFIG_ACCEPT_LIST_SIZE				1
-#define SL_BT_CONFIG_MAX_CONNECTIONS				1
-#define SL_BT_CONFIG_USER_ADVERTISERS				1
-#define SL_BT_CONTROLLER_BUFFER_MEMORY				CONFIG_BT_SILABS_EFR32_BUFFER_MEMORY
-#define SL_BT_CONTROLLER_LE_BUFFER_SIZE_MAX			CONFIG_BT_BUF_ACL_TX_COUNT
-#define SL_BT_CONTROLLER_COMPLETED_PACKETS_THRESHOLD		1
-#define SL_BT_CONTROLLER_COMPLETED_PACKETS_EVENTS_TIMEOUT	3
-#define SL_BT_SILABS_LL_STACK_SIZE				1024
-
-static K_KERNEL_STACK_DEFINE(slz_ll_stack, SL_BT_SILABS_LL_STACK_SIZE);
+#if defined(CONFIG_BT_MAX_CONN)
+#define MAX_CONN CONFIG_BT_MAX_CONN
+#else
+#define MAX_CONN 0
+#endif
+<<<<<<< HEAD
+=======
```

Comment:
I missed this since I only looked at the final diff. Please get rid of this kind of incorrect stuff from the intermediate commits.